thorlcr/master/thgraphmanager.cpp

```diff
@@ -327,13 +327,16 @@ void CJobManager::fatal(IException *e)
 
 void CJobManager::updateWorkUnitLog(IWorkUnit &workunit)
 {
-    StringBuffer log, logUrl;
+    StringBuffer log, logUrl, slaveLogPattern;
     logHandler->getLogName(log);
     createUNCFilename(log, logUrl, false);
-    workunit.addProcess("Thor", globals->queryProp("@name"), 0, logUrl.str());
-}
-
 
+    //Not sure whether this is the correct way to set the slaveLogPattern or not:
```

Comment:
it may be ok. The proper way is to probably to use the IComponentLogFileCreator interface, which the logHandler came from and use the date + thorslave string to create a filename for the slave log.

Reply:
I guess that many changes is needed to use the IComponentLogFileCreator interface. How about leave it as is for now?

Reply:
yes, I'd leave this as is for now.